### ISSUE TITLE
  [BUG] fmDNS: warning in configuration when I preview config

### ISSUE THREAD

**Issue by beza2000** [WillyXJ/facileManager#385]
Replace everything between stars with current version of your facileManager and module installations:  
fM Version : 3.0.1
fmDNS Version : 3.0.2

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---
When I use Keys, I expect Key Name don't change.
But when I previewed config, I had error after named-checkconf - Key Name ended with a dot:
----
```
Your named configuration contains one or more errors:
/etc/named.conf:94: unknown key 'rndc-key'
---
====== /etc/named/zones/views.conf.internal.keys:
1 // This file was built using fmDNS 3.0.1 on Wed, 02 Aug 2017 16:04:28 +0300 Europe/Moscow
2 
3 key "rndc-key." {
4 	algorithm hmac-md5;
5 	secret "U0gwg+QNPnM6IlS/jCWLXw==";
6 };
======

====== /etc/named.conf:
 93 controls {
 94 	inet * allow { localhost; } keys { "rndc-key"; };
 95 };
======
```
---

**Comment by WillyXJ** [comment]
Confirmed this behavior only exists when a key is associated with a view.

**Comment by WillyXJ** [comment]
This is now fixed in [3.0.3](http://www.facilemanager.com/modules/fmdns/) and later.
